Automattic/abacus
# Prevent horizontal scrolling in the experiment results table

Related to #677

This PR is changing how we display the metric name to prevent the horizontal scroll in the Metric Assignment Results table
See https://github.com/Automattic/abacus/issues/677#issuecomment-1136461669 for details.

<img width="570" alt="Screenshot 2022-05-25 at 16 30 39" src="https://user-images.githubusercontent.com/14192054/170305275-7cb665da-6762-4167-ad66-8cad6295e758.png">





## How has this been tested?






- Automated tests that cover added/changed functionality

- Manual testing instructions (please specify):
  - Option 1: Test with an experiment containing metrics with long names such as `20365`
  - Option 2: Run `npm run storybook` and check out `/path=/story/experiment-results--some-analyses-with-long-metric-name`
